tools/git-hooks/pre-commit

```diff
@@ -30,6 +30,13 @@ for FILE in `git diff --name-only` ; do
             ERRORS=true
             echo -e "[ERROR] $FILE: Found pdb.set_trace() debug statements in file.\n"
         fi
+    elif [[ "$FILE" =~ ^.+(js|jsx|ts|tsx)$ ]]; then
```

Comment:
the `npm run lint` only checks files in certain folders I think it'd be good to restrict this hook to only check for changes in those folders (see package.json)

Reply:
Added Folders check